crates/rethnet_rpc_server/src/lib.rs

```diff
@@ -284,6 +294,51 @@ async fn handle_coinbase(state: StateType) -> ResponseData<Address> {
     }
 }
 
+async fn handle_evm_increase_time(
+    state: StateType,
+    increment: U256OrUsize,
+) -> ResponseData<String> {
+    event!(Level::INFO, "evm_increaseTime({increment:?})");
+    let increment: U256 = increment.into();
```

Comment:
Can this be turned into `U256::from(increment)` without getting the same compiler error?

Reply:
No, it would cause that same error, but since implementing `TryFrom<_> for U256` in [this commit](https://github.com/NomicFoundation/hardhat/pull/4227/commits/14e602714da7bd480d2a1071990c568ef0325811) I was able to convert this instance and a few others into `U256::from(_)`.